### DISCUSSION TITLE
maybe this project can be streamlit in rust

### DISCUSSION THREAD

**Discussion by dawsongzhao0523** [sqlpage/SQLPage#15] [Ideas]
is there plan to add more component like streamlit？

**Comment by lovasoa** [comment]
Hi! Is there a component in particular that you would need and you feel is missing ? 

This project could be compared to a streamlit for SQL, not rust. It is implemented in rust, but that is totally transparent for the user. All the user sees and interacts with is SQL, not rust.

**Comment by lovasoa** [comment]
I will move this to the discussions forum, where it is easier to discuss ideas around the project then in issues.
